contentwarehouse/snippets/delete_document_sample.py

```diff
@@ -0,0 +1,52 @@
+# # Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_delete_document]
+
+from google.cloud import contentwarehouse_v1 as contentwarehouse
```

Comment:
You can probably make all of these just:

```suggestion
from google.cloud import contentwarehouse
```

Should default to v1

Reply:
I tried it but it was throwing error that's why I had to used the v1 alias

Reply:
Ah, okay. I don't think need to use the `.types` moniker for the type checking. I'll make suggestions to change these.